### PR TITLE
Accounts Reference management

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR is about a module for handling accounts reference.

An accounts reference is a kind of group of accounts that we can get financial information such as debit, credit and balance for a given period.

In this PR we have : 
- A module for handling crud operation for accounts reference
- An api for having debit, credit, and balance of an accounts reference or for all accounts references.
- Integration tests
- E2E tests

### CODE REVIEW

**Comment by mbayopanda** [comment]
@jniles

File: client/src/i18n/en/form.json [hunk 1]
```diff
@@ -710,6 +710,7 @@
             "EMPLOYEE": "Select an Employee",
             "EMPLOYEE_CONFIGURATION": "Select Employee Configuration",
             "ACCOUNT": "Select an Account",
+            "ACCOUNTS": "Sélect Accounts",
```
Comment (lomamech) [inline client/src/i18n/en/form.json:713]: Try to correct the translation "é"

File: client/src/i18n/fr/account.json [hunk 2]
```diff
@@ -52,6 +52,21 @@
     "CONFIRM_UNHIDE" : "Ce compte est actuellement caché. Êtes-vous sûr de vouloir le révéler pour l'utiliser ?",
     "CONFIRM_LOCK" : "Le verrouillage d'un compte limite la possibilité de publier des transactions en utilisant le compte. Êtes-vous sûr de vouloir le verrouiller ?",
     "CONFIRM_UNLOCK" : "Le compte est bloqué. Le débloquer permettra d'afficher des transactions contenant ce compte. Êtes-vous sûr de vouloir le débloquer ?",
+    "REFERENCE":{
+      "REFERENCE":"Réference",
```
Comment (lomamech) [inline client/src/i18n/fr/account.json]: The word reference has two "é" référence

File: client/src/i18n/fr/tree.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "TREE": {
     "ACCOUNT":"Comptes",
+    "ACCOUNT_REFERENCE_MANAGEMENT":"References des comptes",
```
Comment (lomamech) [inline client/src/i18n/fr/tree.json]: The word reference has two "é" référence

**Review by lomamech — COMMENTED** [review]
Trying to arrange the conversion for the reference word, I'm only reassured that the hidden and blocked accounts do not appear in the list of accounts

**Comment by mbayopanda** [comment]
@lomamech could you bors this PR please

**Comment by lomamech** [comment]
@mbayopanda Try to update this PR

**Pull request merged**